.gitignore

```diff
@@ -11,3 +11,6 @@ Cargo.lock
 
 # These are failure persistance files generated by proptest
 proptest-regressions/
+
+# This is a file generated by the build process:
+stdlib/assets/std.masl
```

Comment:
currently, we run `stdlib/build.rs` when `stdlib/asm` changes

https://github.com/0xPolygonMiden/miden-vm/blob/bffed62b3cbdf509dc0a9ba0d789247df3213607/stdlib/build.rs#L27

with that, we might end up with outdated `std.masl` for dev environments, as described [here](https://github.com/0xPolygonMiden/miden-vm/issues/620#issuecomment-1374821817).

can we also introduce a mechanism on `build.rs` to change not only when `stdlib/asm` changes, but also when the last commit is changed? this would cover all discussed cases.

Reply:
As note: I don't know if that is currently possible. I'm checking the documentation, and I don't see such option. We can, instead, introduce a watcher for the only build-dependency of `stdlib`: 

`println!("cargo:rerun-if-changed=../assembly/src");`

Reply:
Sounds good, I can add the watcher to the PR.